File: rb/utility/prepare_body.rb
# NeugeborenenVornamenKantonStgallen SDK utility: prepare_body
module NeugeborenenVornamenKantonStgallenUtilities
  PrepareBody = ->(ctx) {
    ctx.op.input == "data" ? ctx.utility.transform_request.call(ctx) : nil
  }
end

File: .sdk/tm/php/utility/PrepareBody.php
<?php
declare(strict_types=1);

// NeugeborenenVornamenKantonStgallen SDK utility: prepare_body

class NeugeborenenVornamenKantonStgallenPrepareBody
{
    public static function call(NeugeborenenVornamenKantonStgallenContext $ctx): mixed
    {
        if ($ctx->op->input === 'data') {
            return ($ctx->utility->transform_request)($ctx);
        }
        return null;
    }
}
